lib/es-models/bib.js

```diff
@@ -807,6 +809,8 @@ class EsBib extends EsBase {
   }
 
   _isAeonUrl (url) {
+    if (typeof url !== 'string') return false
```

Comment:
is this a case that you've seen or just extra caution?

Reply:
Should now be protected upstream, but good to check types before calling 'startsWith'